### ISSUE TITLE
Set id to feature

### ISSUE THREAD

**Issue by korywka** [maplibre/martin#65]
Is there a way to set `id` to feature (last param of `ST_AsMVT` function)?

`id` from table is not passed to feature id.

![image](https://user-images.githubusercontent.com/988471/76747965-a92e1380-6782-11ea-8d24-002f632cf058.png)

Or int value:

![image](https://user-images.githubusercontent.com/988471/76750031-3757c900-6786-11ea-8068-877c140fb1cc.png)

**Comment by AdriSolid** [comment]
Hi! You can do it in your frontend as well (Mapbox GL JS). Call your id (int or string) on the function source and then:

```javacript
map.addSource('counties', {
  "type": "vector",
  "url" || "tiles": "your url / tiles",
  "promoteId": "your id"
});

**Comment by korywka** [comment]
@AdriSolid For now I do it with custom function and `ST_AsMVT` (https://postgis.net/docs/ST_AsMVT.html) takes last param as feature_id_name, but your advice is very helpful too. Thanks! 
Will leave doc link here for the community: https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/#vector-promoteId

**Comment by shaunakv1** [comment]
great idea @AdriSolid but that is a very Mapbox GL specific feature. I do, think this feature is very valuable to have in API itself that way it's available agnostic of which client is being used to render the tiles.

**Comment by nyurik** [comment]
This is partially already implemented, and will be cleaned up and fixed in #380
